plugin/helpers.go

```diff
@@ -29,8 +33,23 @@ type Helpers interface {
 	//
 	// Minimum server version: 5.6
 	KVSetWithExpiryJSON(key string, value interface{}, expireInSeconds int64) error
+
+	// RegisterCommand registers the given command and callback for the ExecuteCommand helper.
+	// Returns error if the callback is not provided or the API.RegisterCommand fails
```

Comment:
nit: I maintain we don't need to describe the error conditions here, since they aren't "unusual".
```suggestion
```